src/content/peerconnection/pc1/js/main.js

```diff
@@ -68,6 +71,7 @@ function gotStream(stream) {
 
 function start() {
   trace('Requesting local stream');
+  webAudio.start();
```

Comment:
You need to add a start() method (or am I missing something?)

Example here: https://github.com/webrtc/samples/blob/gh-pages/src/content/peerconnection/webaudio-input/js/webaudioextended.js